src/test/passwordRest.spec.js

```diff
@@ -0,0 +1,83 @@
+import chai, { expect } from 'chai';
+import chaiHttp from 'chai-http';
+import app from '../index';
+import TestHelper from '../utils/testHelper';
+import db from '../models';
+
+chai.use(chaiHttp);
+chai.should();
+
+
+describe('/api/v1/auth', () => {
+  let passwordResetToken;
+
+  before((done) => {
+    TestHelper.destroyModel('User');
+    done();
+  });
+  describe('POST /forgot_password', () => {
+    before(async () => {
+      await db.User.create({
+        first_name: 'irellevant',
+        last_name: 'Tester',
+        email: 'testa@test.com',
+        password: 'PasswordTest123'
+      });
+    });
+    it('should return an error for an unregistered user', async () => {
```

Comment:
leave a line after line 27

Reply:
Eslint flags it when I try to leave a space, should I disable it?

Reply:
I am not sure of that because I gave this feedback to other members of the team and they implemented without Eslint complaining

Reply:
I just pulled your branch to my local system and tried it but Eslint didn't complain